server/channels/store/sqlstore/group_store.go

```diff
@@ -215,7 +215,12 @@ func (s *SqlGroupStore) buildInsertGroupUsersQuery(groupId string, userIds []str
 
 func (s *SqlGroupStore) Get(groupId string) (*model.Group, error) {
 	var group model.Group
-	if err := s.GetReplicaX().Get(&group, "SELECT * from UserGroups WHERE Id = ?", groupId); err != nil {
+	query, args, _ := s.getQueryBuilder().
```

Comment:
What was the reason of ignoring errors here?

Reply:
Oops, looks like I forgot to capture the error values 😅
I will capture them and add in early returns if an error is found.